structures.h

```diff
@@ -0,0 +1,74 @@
+typedef struct {
+    int day, month, year;
+} date;
+
+
+typedef struct {
+    char name[255], surname[255];
+    char sex[255];
+    date birthday;
+} Person;
+
+
+typedef struct {
+    char name[255];
+    double price;
+} Equipment;
+
+
+typedef struct {
+    char name[255];
+    double price;
+    Equipment equipment; // ??? equipment in product or product in equipment ???
+} Product;
+
+
+typedef struct {
+    char name[255];
+    double wage_rate;
+    Equipment equipment;
+} Profession;
+
+
+typedef struct {
+    Person person;
+    Profession profession;
+    double salary;
+    date employed;
+    double experience;
+    double product_rate;
+} Employee;
+
+
+typedef struct {
+    Employee employee;
+} Deputy;
+
+
+typedef struct {
+    Employee employee;
+} Head;
+
+
+
+
+date inputDate();

```

Comment:
It is much better than it was.
However U'd better split not by functionality, but by classes: Empoyee.h. Empoyee.c, Product.h, Product.c etc